interfaces/builtin/docker_test.go

```diff
@@ -63,20 +70,20 @@ func (s *DockerInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	snippet, err := s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecurityAppArmor)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
-	// connected plugs have a non-nil security snippet for seccomp
-	snippet, err = s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecuritySecComp)
```

Comment:
It looks like we are loosing a test here - but given that its tested (properly) below in TestConnectedPlugSnippet it seems like this test was redundant in the first place.